.github/workflows/aws-lc-rs.yml

```diff
@@ -10,67 +10,120 @@ concurrency:
 env:
   GOPROXY: https://proxy.golang.org,direct
   AWS_LC_SYS_CMAKE_BUILDER: 1
-  RUST_NIGHTLY_TOOLCHAIN: nightly-2024-05-22
+  RUST_NIGHTLY_TOOLCHAIN: nightly
+  RUST_SCRIPT_NIGHTLY_TOOLCHAIN: nightly-2024-05-22
```

Comment:
considering it's been a while, are we able to update this now?

Reply:
We could update it, but we'd have to also change our scripts b/c they are hard-coded to this particular toolchain: https://github.com/aws/aws-lc-rs/blob/main/scripts/tools/semver.rs#L1
```
#!/usr/bin/env -S cargo +nightly-2024-05-22 -Zscript
...
```

Reply:
probably in another PR then

Reply:
Working on it now: https://github.com/aws/aws-lc-rs/pull/641